### PR TITLE
[PDI-19290]- Added Validation checks for "Nr. of packets to send" 

### PR DESCRIPTION
**Pull request by AAravind27**
Added Validation checks for "Nr. of packets to send" in Ping a host. In UI, Entering Any non-numeric value (and negative integers) will show a message asking user to enter numeric values or positive numbers.
Incase, if a kjb/ktr already has non-numeric value (and negative integers) ,value will be replaced with
2( i.e. default value  ) during runtime and user will be notified about this replacement in logs.

### CODE REVIEW

**Comment by buildguy** [comment]
### :warning: Build finished in 31m 14s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl engine,ui
```
---
### :no_entry: Failed Tests

#### :cloud_with_lightning_and_rain: 1 test(s) failed:

<details><summary><b>org.pentaho.di.trans.steps.avro.input.AvroInputMetaTest.testGetFields_unknownPluginForFieldType</b> (click to expand)</summary><p>

```
Expected exception: org.pentaho.di.core.exception.KettleStepException
```

</p></details>

  
Tests run: **3695**, Failures: **1**, Skipped: **4** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pdi-11/job/PR-8756/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: engine/src/main/java/org/pentaho/di/job/entries/ping/JobEntryPing.java [hunk 1]
```diff
@@ -198,7 +198,26 @@ public void saveRep( Repository rep, IMetaStore metaStore, ObjectId id_job ) thr
     }
   }
 
+  public static boolean isNumeric( String strNum ) {
```
Comment (peterrinehart) [inline engine/src/main/java/org/pentaho/di/job/entries/ping/JobEntryPing.java:201]: If similar functionality is needed in the dialog also, we should try to make this a method used between both somehow.  
Also, we should add a unit test for the new logic, at minimum (and/or make an integration test ensuring the step behaves as desired when provided with bad data).
